fitnessllm_dataplatform/task_handler.py

```
@@ -0,0 +1,55 @@
+
+        Raises:
+            KeyError: If required options or environment variables are missing.
+            ValueError: If data source is not supported.
+        """
+        if data_source not in [member.value for member in FitnessLLMDataSource]:
+            raise ValueError(f"Unsupported data source: {data_source}")
+        strava_api_interface = StravaAPIInterface(self.InfrastructureNames)
+        try:
+            strava_api_interface.get_all_data()
+        except Exception as e:
+            raise RuntimeError(f"Failed to get data from Strava API: {e}") from e
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)